services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -6295,6 +6296,7 @@ private void waitUntilNodesAreAssignedForResource(
         ResourceAssignment resourceAssignment = routingDataRepository.getResourceAssignment();
         notReadyReason =
             statusDecider.hasEnoughNodesToStartPush(topic, replicationFactor, resourceAssignment, notReadyReason);
+        elapsedTime = System.currentTimeMillis() - startTime;
         if (!notReadyReason.isPresent()) {
           LOGGER.info("After waiting for {}ms, resource allocation is completed for: {}.", elapsedTime, topic);
```

Comment:
Maybe we only need this log line change
`LOGGER.info("After waiting for {}ms, resource allocation is completed for: {}.", (System.currentTimeMillis() - startTime), topic);`